Formula/arx-libertatis.rb

```diff
@@ -25,6 +25,23 @@ class ArxLibertatis < Formula
   def install
     args = std_cmake_args
 
+    # The patches for these aren't straightforward to backport because of
+    # other changes; these minimal inreplaces get it building.
+    # HEAD is fine, and the next stable release will contain these changes.
```

Comment:
The last stable release was in 2013. It would be good to get some kind of commitment out of upstream to at least create a new tag at some point soon.